stored_requests/backends/http_fetcher/fetcher.go

```diff
@@ -80,8 +81,45 @@ func (fetcher *HttpFetcher) FetchRequests(ctx context.Context, requestIDs []stri
 	return
 }
 
-func (fetcher *HttpFetcher) FetchCategories(primaryAdServer, publisherId, iabCategory string) (string, error) {
-	return "", nil
+func (fetcher *HttpFetcher) FetchCategories(ctx context.Context, primaryAdServer, publisherId, iabCategory string) (string, error) {
+	if fetcher.Categories == nil {
+		fetcher.Categories = make(map[string]map[string]stored_requests.Category)
+	}
+
+	if publisherId == "" {
+		publisherId = primaryAdServer
```

Comment:
This means we could have something like `freewheel_freewheel` as the `dataName` below. Is this correct, or should it resolve to just `freewheel`?

Reply:
You're right, fixed!